doc/manual/source/utilities/toolsutility.rst

```diff
@@ -0,0 +1,16 @@
+.. _Tools Utility:
+
+:tocdepth: 2
+
+Tools Utility
+#############
+
+The ``diffpy.utils.tools`` module is designed for users to manage and track their username and email information.
```

Comment:
the tools package contains various tool functions. for use with diffpy apps.   One of these "<name>" handles the username and email.    I expect new tools will be added in the future.